packages/components/src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import css from '@styled-system/css';
+import { User } from '@codesandbox/common/lib/types';
+import styled from 'styled-components';
+import { Element } from '../Element';
+import { Text } from '../Text';
+
+interface IAvatarProps {
+  user: User;
+}
+
+export const AvatarImage = styled.img(
+  css({
+    width: 8,
+    height: 8,
+    borderRadius: 'small',
+    border: '1px solid rgba(255, 255, 255, 0.46)',
+    backgroundColor: 'rgba(255, 255, 255, 0.46)', // fallback for loading image
+  })
+);
+
+export const Pro = styled(Text).attrs({ size: 1, weight: 'bold' })(
+  css({
+    backgroundColor: 'blues.700',
+    color: 'sideBarTitle.foreground',
+    borderRadius: 'small',
+    paddingX: 2,
+    paddingY: '2px', // no 2 in spaces
+    border: '1px solid',
+    borderColor: 'sideBar.border',
```

Comment:
border on pro = sidebar.border sounds like the perfect use case for polyfill layer.

Reply:
so fuck the rgba?